quickstart/gcp/tutorial-gke.md

```diff
@@ -401,3 +377,7 @@ Managed Kubernetes cluster on GCP GKE.
 
 For a follow-up example on how to use Pulumi programs to create a Kubernetes
 apps on your new cluster, see [Kubernetes Tutorial: Getting Started With Pulumi](../kubernetes/tutorial-configmap-rollout.html).
+
+We also encourage you to watch Joe Beda, co-founder of Kubernetes and Heptio,
+[take Pulumi for a spin on GKE](https://youtu.be/ILMK65YVSKw) in an
+episode of [TGIK8s](https://github.com/heptio/tgik).
```

Comment:
Should we just embed the YouTube video inline here.

Reply:
Embedding [doesn't seem possible](https://stackoverflow.com/a/16079387/2274669) in markdown. Do you have a suggestion?

Reply:
I think you can just include the inline HTML.  For example: https://stackoverflow.com/questions/10529859/how-to-include-video-in-jekyll-markdown-blog

Reply:
Ah ok, wasn't aware we used jekyll. I've gone ahead and included the inline HTML in the most recent commit